webrender/src/internal_types.rs

```diff
@@ -335,6 +335,7 @@ pub enum RenderTargetMode {
 pub enum TextureUpdateDetails {
     Raw,
     Blit(Vec<u8>),
+    BlitWithStride(Vec<u8>, u32),
```

Comment:
Instead of introducing a new type here, could we just add the stride to the Blit enum variant and always expect stride?